GameServerLib/Config.cs

```diff
@@ -90,6 +97,30 @@ private void LoadConfig(Game game, string json)
             MapData = ContentManager.GetMapData(GameConfig.Map);
             MapSpawns = ContentManager.GetMapSpawns(GameConfig.Map);
         }
+
+        private string getContentPath()
+        {
+            string result = null;
+            var path = new DirectoryInfo(Directory.GetCurrentDirectory());
+
+            while (result == null) {
```

Comment:
This can be an infinite loop (there are users that somehow delete their Content folder), use some counter to prevent this (depth >8 or so shouldn't be allowed).

Reply:
As we're using getParent() and C:\ does not have a Parent anymore, this should be fine. (path = null  -> break)

Reply:
Sounds fine. I mark it as resolved.